### PR TITLE
Restore deleted method.

### PR DESCRIPTION
**Pull request by anuraaga**
Realized I accidentally deleted this method in #1940

### CODE REVIEW

File: core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointGroup.java [hunk 1]
```diff
@@ -50,7 +52,7 @@ static EndpointGroup empty() {
 
     /**
      * Waits until the initial {@link Endpoint}s are ready.
-     *
+     *2
```
Comment (minwoox) [inline core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointGroup.java]: typo here?

File: core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointGroup.java [hunk 2]
```diff
@@ -62,6 +64,22 @@ static EndpointGroup empty() {
         }
     }
 
+    /**
+     * Waits until the initial {@link Endpoint}s are ready, with timeout.
+     *
+     * @throws java.util.concurrent.CancellationException if {@link #close()} was called before the initial
+     * {@link Endpoint}s are set
```
Comment (minwoox) [inline core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointGroup.java]: indentation?

File: core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointGroup.java [hunk 3]
```diff
@@ -52,7 +54,7 @@ static EndpointGroup empty() {
      * Waits until the initial {@link Endpoint}s are ready.
      *
      * @throws java.util.concurrent.CancellationException if {@link #close()} was called before the initial
-     * {@link Endpoint}s are set
+     *     {@link Endpoint}s are set
```
Comment (ikhoon) [inline core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointGroup.java]: remove white space?

Comment (anuraaga) [reply core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointGroup.java]: @minwoox  asked to indent :) Admittedly I'm not sure what the indentation rules of javadoc are

Comment (trustin) [reply core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointGroup.java]:
```
@param myParam description first line
               second line
@throws FooException description first line
                     second line
```

..?

Comment (minwoox) [reply core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointGroup.java]: Sorry about the comment that I left. It was unclear.
I think we need to align the `{` under the `if`. :)

File: core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointGroup.java [hunk 4]
```diff
@@ -62,6 +64,22 @@ static EndpointGroup empty() {
         }
     }
 
+    /**
+     * Waits until the initial {@link Endpoint}s are ready, with timeout.
+     *
+     * @throws java.util.concurrent.CancellationException if {@link #close()} was called before the initial
```
Comment (trustin) [inline core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointGroup.java]: Could use an import to give more room for the description:

```
@throws CancellationException if {@link #close()} ...
```

**Pull request merged**